.github/workflows/ci.yml

```
@@ -245,6 +245,12 @@ jobs:
       # - name: Install M2Crypto (py2.6)
       #   if: ${{ matrix.container }}
       #   run: yum install -y m2crypto
+      - name: Install kyber-py on py3.9 and later
+        if: ${{ !matrix.container && matrix.python-version != '2.6' && matrix.python-version != '2.7' && matrix.python-version != '3.6' && matrix.python-version != '3.7' && matrix.python-version != '3.8' }}
+        run: |
+          wget https://github.com/GiacomoPope/kyber-py/archive/refs/tags/v1.0.zip
+          mv v1.0.zip kyber_py-1.0.zip
```

nit: we could avoid downloading and moving by saying the wget to store the file as we need.
```suggestion
          wget https://github.com/GiacomoPope/kyber-py/archive/refs/tags/v1.0.zip -O kyber_py-1.0.zip
```